packages/@react-types/calendar/src/index.d.ts

```diff
@@ -50,21 +50,29 @@ export interface CalendarPropsBase {
   /** Handler that is called when the focused date changes. */
   onFocusChange?: (date: CalendarDate) => void,
   /** Whether the current selection is valid or invalid according to application logic. */
-  validationState?: ValidationState,
-  /** An error message to display when the selected value is invalid. */
-  errorMessage?: ReactNode
+  validationState?: ValidationState
 }
 
 export type DateRange = RangeValue<DateValue>;
-export interface CalendarProps<T extends DateValue> extends CalendarPropsBase, ValueBase<T, MappedDateValue<T>>, DOMProps, AriaLabelingProps {}
-export interface RangeCalendarProps<T extends DateValue> extends CalendarPropsBase, ValueBase<RangeValue<T>, RangeValue<MappedDateValue<T>>>, DOMProps, AriaLabelingProps {
+export interface CalendarProps<T extends DateValue> extends CalendarPropsBase, ValueBase<T, MappedDateValue<T>> {}
+export interface RangeCalendarProps<T extends DateValue> extends CalendarPropsBase, ValueBase<RangeValue<T>, RangeValue<MappedDateValue<T>>> {
   /**
    * When combined with `isDateUnavailable`, determines whether non-contiguous ranges,
    * i.e. ranges containing unavailable dates, may be selected.
    */
   allowsNonContiguousRanges?: boolean
 }
 
+export interface AriaCalendarProps<T extends DateValue> extends CalendarProps<T>, DOMProps, AriaLabelingProps {
+  /** An error message to display when the selected value is invalid. */
+  errorMessage?: ReactNode
```

Comment:
Unsure about this one, I think it is only used in the aria hooks. If so, I'll apply the same change to DatePicker/etc types